### PR TITLE
OG contract

### PR DESCRIPTION
**Pull request by Prannav-k**

### CODE REVIEW

File: app/migrations/init_db.py [hunk 1]
```diff
@@ -272,8 +272,8 @@ def init_db():
                     )
                     ''')
     cur.execute('''
-                    CREATE UNIQUE INDEX IF NOT EXISTS idx_stake_ledger_wallet 
-                    ON stake_ledger (wallet_address)
+                    CREATE UNIQUE INDEX IF NOT EXISTS idx_stake_ledger_address_person_id
+                    ON stake_ledger (address,person_id)
```
Comment (kousthubraja) [inline app/migrations/init_db.py:276]: @Prannav-k one wallet can have only one entry in stake ledger right?

File: app/codes/helpers/TransactionCreator.py [hunk 2]
```diff
@@ -96,7 +97,9 @@ def transaction_type_3(self, data: dict):
                 "block_index": 0,
                 "specific_data": txspecdata
             },
-            "signatures": []
+            "signatures": [{
```
Comment (kousthubraja) [inline app/codes/helpers/TransactionCreator.py:100]: why is this necessary?

File: app/codes/contracts/og_issue.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        required_value = {
+            "token_code": issuance_token_code,
+            "amount": amount_to_issue
+        }
+
+        if required_value in value:
+            '''txn type 5 (one way transfer)'''
+            transaction_creator = TransactionCreator()
+            transfer_proposal_data = {
+                "transfer_type": 1,
```
Comment (kousthubraja) [inline app/codes/contracts/og_issue.py:81]: is `transfer_type` allowed?

**Review by kousthubraja — COMMENTED** [review]
yet to fully review. still going on. you can test this in local.

File: app/migrations/init_db.py (hunk 1 above)
Comment (Prannav-k) [reply app/migrations/init_db.py:276]: yes , for a contract , one wallet can have only one entry. But there can be 2 contracts (newrl stake and og stake), so we need contract address + pid/wallet combination.
